build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1645,6 +1645,29 @@ message UpdateActionResultRequest {
   // length of the action digest hash and the digest functions announced
   // in the server's capabilities.
   DigestFunction.Value digest_function = 5;
+
+  enum Overwrite {
+    // The server may choose to follow either OVERWRITE or NOOVERWRITE
+    // semantics. This option is included for backward compatibility with
+    // earlier API versions.
+    UNSPECIFIED = 0;
+    // On a successful call, the server MUST write the provided ActionResult
+    // to the cache and use that ActionResult as the return value, even if
+    // another cache result already exists. Overwriting eliminates the need
+    // for the client to download alternate outputs, but can cause cache
+    // thrashing on downstream Actions.
+    OVERWRITE = 1;
+    // On a successful call, if a preexisting cache result already exists,
+    // the server MUST ignore the provided ActionResult and return the existing
+    // one instead. If a preexisting cache result does not exist, the server
+    // MUST write the provided ActionResult to the cache and use that
+    // ActionResult as the return value.
+    NOOVERWRITE = 2;
```

Comment:
Should this be `NO_OVERWRITE` for consistency with other enum values?

Reply:
Done